### PR TITLE
6.3.0 Plugin Release Notes

### PR DESCRIPTION
**Pull request by jamesesilvia**
## Description
Update plugin release notes for 6.3.0 (IBU),

## Testing
Manually view each plugin page

### CODE REVIEW

File: docs/plugin_cloud_ha.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ The internal state machines wait for the value set in seconds in `up-holddown-ti
 
 The internal state machines wait for the `peer-reachability-timeout` after every health status before considering the node unreachable. The timer is reset if a health report is processed before the timeout expires.
 
-When the HA Agent determines that a node must become active, the first active redundant interface's mac address and the list of configured prefixes are sent to the API Agent's appropriate REST endpoint. The HA Agent changes the provisional status of the configured `redundant-interface` to **Up**. 
+When the HA Agent determines that a node must become active, the first active redundant interface's mac address and the list of configured prefixes are sent to the API Agent's appropriate REST endpoint. The HA Agent changes the provisional status of the configured `redundant-interface` to **Up**.
```
Comment (Chr1st0ph3rTurn3r) [inline docs/plugin_cloud_ha.md:52]:
```suggestion
When the HA Agent determines that a node must become active, the first active redundant interface's MAC address and the list of configured prefixes are sent to the API Agent's appropriate REST endpoint. The HA Agent changes the provisional status of the configured `redundant-interface` to **Up**.
```

File: docs/plugin_cloud_ha.md [hunk 2]
```diff
@@ -312,7 +312,7 @@ To facilitate the transport of the health status between members of the same gro
 
 ### Azure Load Balancer Config Generation
 
-If the `solution-type` of a group is `azure-lb`, then the plugin will generate services and service routes to route the http probes down into the `cloud-ha` interface for the Azure Load Balancer API Agent to respond to. 
+If the `solution-type` of a group is `azure-lb`, then the plugin will generate services and service routes to route the http probes down into the `cloud-ha` interface for the Azure Load Balancer API Agent to respond to.
```
Comment (Chr1st0ph3rTurn3r) [inline docs/plugin_cloud_ha.md:315]:
```suggestion
If the `solution-type` of a group is `azure-lb`, then the plugin will generate services and service routes to route the http probes down into the `cloud-ha` interface for the Azure Load Balancer API Agent to respond.
```

File: docs/plugin_cloud_ha.md [hunk 3]
```diff
@@ -970,7 +970,7 @@ exit
 Below is a sample running configuration for a node and a service.
 
 :::info
-Use the following configuration as an example only - it should not be used on a system as is. 
+Use the following configuration as an example only - it should not be used on a system as is.
```
Comment (Chr1st0ph3rTurn3r) [inline docs/plugin_cloud_ha.md:973]:
```suggestion
The following configuration is an example only - It is not for use on a system.
```

File: docs/plugin_loopback_static_routes.md [hunk 4]
```diff
@@ -2,6 +2,10 @@
 title: Loopback Static Routes Plugin
 sidebar_label: Loopback Static Routes
 ---
+:::important
+This plugin has been deprecated starting in SSR 6.3.0.
```
Comment (Chr1st0ph3rTurn3r) [inline docs/plugin_loopback_static_routes.md:6]:
```suggestion
This plugin has been deprecated with the release of SSR 6.3.0.
```

File: docs/plugin_mosh.md [hunk 5]
```diff
@@ -5,6 +5,10 @@ sidebar_label: Mosh
 <!-- markdown-link-check-disable-next-line -->
 The Mosh plugin provides the ability to install the MObile SHell [MOSH](https://mosh.org/) on SSR nodes. Mosh is provided under GNU GPLv3.
 
+:::important
+This plugin has been deprecated starting in SSR 6.3.0.
```
Comment (Chr1st0ph3rTurn3r) [inline docs/plugin_mosh.md:9]:
```suggestion
This plugin has been deprecated with the release of SSR 6.3.0.
```

File: docs/plugin_set_hostname.md [hunk 6]
```diff
@@ -2,6 +2,10 @@
 title: Set Hostname
 ---
 
+:::important
+This plugin has been deprecated starting in SSR 6.3.0.
```
Comment (Chr1st0ph3rTurn3r) [inline docs/plugin_set_hostname.md:6]:
```suggestion
This plugin has been deprecated with the release of SSR 6.3.0.
```

**Review by Chr1st0ph3rTurn3r — CHANGES_REQUESTED** [review]

File: docs/_ha_sync_plugin_release_notes.md [hunk 7]
```diff
@@ -3,6 +3,10 @@
 The plugin must be updated to version 1.1.0 or later prior to [upgrading the conductor to SSR version 5.4.0.](intro_upgrade_considerations.md#plugin-configuration-generation-changes)
 :::
 
+### Release 2.0.0
+
+Image based install and upgrade (IBU) support for SSR 6.3.0.
+
```
Comment (MichaelBaj) [inline docs/_ha_sync_plugin_release_notes.md:9]: Other files include:

**Release Date:** Sep 30, 2024

Was this accidentally omitted?

Comment (jamesesilvia) [reply docs/_ha_sync_plugin_release_notes.md:9]: Yes, let me fix

**Review by MichaelBaj — APPROVED** [review]

**Review by Chr1st0ph3rTurn3r — APPROVED** [review]

File: docs/plugin_cloud_ha.md [hunk 8]
```diff
@@ -312,7 +312,7 @@ To facilitate the transport of the health status between members of the same gro
 
 ### Azure Load Balancer Config Generation
 
-If the `solution-type` of a group is `azure-lb`, then the plugin will generate services and service routes to route the http probes down into the `cloud-ha` interface for the Azure Load Balancer API Agent to respond to. 
+If the `solution-type` of a group is `azure-lb`, then the plugin will generate services and service routes to route the http probes down into the `cloud-ha` interface for the Azure Load Balancer API Agent to respond.
```
Comment (jamesesilvia) [inline docs/plugin_cloud_ha.md:315]: Sorry my editor removes whitespace on save so thats why these spaces/tabs were removed. it shouldnt impact formatting i dont think

**Pull request merged**